src/editors/sharedComponents/BaseModal/index.jsx

```diff
@@ -49,9 +50,11 @@ const BaseModal = ({
       <ActionRow>
         {footerAction}
         <ActionRow.Spacer />
```

Comment:
```suggestion
        {footerAction && (
          <>
            {footerAction}
            <ActionRow.Spacer />
          </>
        )}
```

Removes the blank space before `Discard` button.

Reply:
Update: https://github.com/openedx/frontend-app-authoring/pull/1732/commits/925546935fff485095e2f69df044257bb62bf44a

Reply:
We don't need this change now